use fmt::format while clang doesn't support std::format yet

Index: rts/System/float3.h
--- rts/System/float3.h.orig
+++ rts/System/float3.h
@@ -8,6 +8,8 @@
 #include <utility>
 #include <format>
 
+#include <fmt/core.h>
+
 #include "System/BranchPrediction.h"
 #include "lib/streflop/streflop_cond.h"
 #include "System/creg/creg_cond.h"
@@ -806,7 +808,7 @@ class float3 (public)
 	static constexpr float nrm_eps() { return 1e-12f; }
 
 	std::string str() const {
-		return std::format("float3({:.3f}, {:.3f}, {:.3f})", x, y, z);
+		return fmt::format("float3({:.3f}, {:.3f}, {:.3f})", x, y, z);
 	}
 
 	/**
